ckanext/canada/templates/admin/trash.html

```diff
@@ -8,7 +8,7 @@
       <ul class="user-list list-group">
         {% for pkg in g.deleted_packages %}
           {% set title = pkg.title or pkg.name %}
-          <li class="list-group-item">{{ h.link_to(h.truncate(title, truncate_title), h.url_for(pkg.type + '.read', id=pkg.name)) }}</li>
+          <li class="list-group-item">{{ h.link_to(h.truncate(title, truncate_title), h.url_for(pkg.type + '.read', id=pkg.name), title=title) }}</li>
```

Comment:
What is the value of `truncate_title` in this template, and does it need to be increased?